dist/robots.cfg

```diff
@@ -201,6 +215,14 @@ RobotIP <<EOR
     82.99.30.71,	# Munax
     82.99.30.72,	# Munax
     82.99.30.73,	# Munax
+    185.54.12*, # AlertLogic
+#    114.119*, # Aspeigalbot
+#    66.249*, # Googlebot
+#    54.236*, # Pinterestbot
+#	23.22.*, #amazonbot
+#    3.224.*, #amazonbot
+#    52.70.*, #amazonbot Trying just to UA now that we removed GECKO and MSIE
```

Comment:
Can we just delete these since they're all commented out?

Reply:
Done now, sorry, thought I did it in the first pass.